library/core/src/intrinsics/mir.rs

```diff
@@ -249,6 +249,29 @@
 //!  - [`Call`] has an associated function as well. The third argument of this function is a normal
 //!    function call expression, for example `my_other_function(a, 5)`.
 //!
+//! #### Debuginfo
+//!
+//!  - A debuginfo name can be given to a local using `debug my_name => contents;`.
+//!    For `contents`, we use the same syntax as operands, to support both places and constants.
```

Comment:
````suggestion
Debuginfo associates source code variable names with MIR expressions that indicate where the value of that variable is stored. The syntax to do so is:
```
debug source_var_name => expression;
```
For `expression`, we use the same syntax as operands, to support both places and constants.
````